boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts

```diff
@@ -119,7 +124,7 @@
 		};
 		storage_partition: partition@f0000 {
 			label = "storage";
-			reg = <0x000f0000 0x00010000>;
+			reg = <0x000f0000 0x00008000>;
```

Comment:
is this related with this change ?

Reply:
No, this is not related to the current PR. This is related to an another Flash driver PR.
I was confused with possible merge conflicts, but now it is clear for me.

Removed this change from the current PR.